client/components/tinymce/plugins/embed/docs/example.jsx

```diff
@@ -45,8 +49,26 @@ export default class EmbedDialogExample extends PureComponent {
 					isVisible={ this.state.showDialog }
 					onCancel={ this.onCancel }
 					onUpdate={ this.onUpdate }
+					siteId={ this.props.siteId }
 				/>
 			</Card>
 		);
 	}
 }
+
+const connectedEmbedDialogExample = connect( state => {
+	return {
+		siteId: get( getCurrentUser( state ), 'primary_blog' ),
+	};
+} )( EmbedDialogExample );
+
+connectedEmbedDialogExample.displayName = 'EmbedDialogExample';
+
+// todo
+// connecting this component feels wrong. it's an example, so shouldn't it instantiate EmbedDialog with renderWithReduxStore
```

Comment:
I tend to agree here.  Are we just providing the siteId here because it's required by some child component or something? I don't think this is a blocker for this PR so we can get this launched, but I think we should do what it takes to disconnect this example from the state tree in a follow-up.